code/modules/antagonists/space_dragon/space_dragon.dm

```diff
@@ -62,14 +62,8 @@
 	return ..()
 
 /datum/antagonist/space_dragon/get_preview_icon()
-	var/icon/icon = icon('icons/mob/nonhuman-player/spacedragon.dmi', "spacedragon")
-
-	icon.Blend(COLOR_STRONG_VIOLET, ICON_MULTIPLY)
-	icon.Blend(icon('icons/mob/nonhuman-player/spacedragon.dmi', "overlay_base"), ICON_OVERLAY)
-
-	icon.Crop(10, 9, 54, 53)
+	var/icon/icon = icon('icons/mob/nonhuman-player/spacedragon_preview.png')
```

Comment:
Please please please do your best to make this something that is generated automatically. We need to avoid these wherever possible because sprites change all the time, and we want it to be literally zero effort for the prefs icons to get updated with them.

Reply:
I would make the dragon overlay a png (then put it with icons/ui_icons/antags), but make the fish in the background dynamically generated.